lib/model/QubitTerm.php

```diff
@@ -336,7 +337,24 @@ public function delete($connection = null)
 
     QubitSearch::getInstance()->delete($this);
 
-    parent::delete($connection);
+    // Replicate parent class delete functionality, skipping nested set updating
```

Comment:
That's a valid take. It's not something we'll commonly need... only thing I can think of is for migrations where you're migrating onto an estisting AtoM instance where data has to be pruned off. Managing patches is a bit of a pain, but yes, this PR could present a pitfall in the future. Maybe I should take another look at modifying the parent class (which would be trivial, except it gets auto-generated by a builder class).